catalyst/utils/config.py

```diff
@@ -1,23 +1,24 @@
-from typing import Any, Dict, List, Union  # isort:skip
 from collections import OrderedDict
 import copy
+from itertools import chain
 import json
-from logging import getLogger
+import logging
 import os
 from pathlib import Path
 import platform
 import re
 import shutil
 import subprocess
 import sys
+from typing import Any, Dict, List, Union
 
 import safitty
 import yaml
 
 from catalyst import utils
 from catalyst.utils.tools.tensorboard import SummaryWriter
 
-LOG = getLogger(__name__)
+logger = logging.getLogger(__name__)
```

Comment:
Do we really need to import logging? Seems `logger` is never used in this module...

Reply:
Initially, I didn't change anything with `logger` in this PR, but I removed this line.